src/OpenLoco/CompanyManager.cpp

```diff
@@ -456,4 +473,14 @@ namespace OpenLoco::CompanyManager
     {
         return competingColourMask(_updating_company_id);
     }
+
+}
+
+namespace OpenLoco
+{
+    CompanyId_t Company::id() const
```

Comment:
While I realise this entire function is temporary, but it feels a bit hacky to declare it here. Unfortunately, it's used quite a bit, so it'll be around for a while yet. I assume it's declared to avoid having to include `CompanyManager.h` in `Company.cpp`?

Reply:
It's done like this as you need to access the raw address the other solution would be to include gamestate.h in company.cpp. it's not the most elegant but i think it's the best solution as i don't really want gamestate.h to be included in a huge amount of files.

Reply:
Ah right, the raw getter function is static. Agreed that we should limit exposing the game state to managers and systems.